From 1b53e87e4f2bd0910e584921c7e18cb97ff13d2a Mon Sep 17 00:00:00 2001
From: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Date: Mon, 20 Nov 2017 11:51:13 +0200
Subject: [PATCH 2842/5242] MLK-16911: drm: imx: dcss: do not advertise
 modifiers

commit  34e6a03c7e1aa2b34032778576ce65cd9f0f974f from
https://source.codeaurora.org/external/imx/linux-imx.git

Since support for tiled formats has not been added to DCSS DRM driver,
do not allow usage of FB modifiers by userspace.

Signed-off-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/dcss/dcss-crtc.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/imx/dcss/dcss-crtc.c b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
index 7cfcc3b..aeafc46 100644
--- a/drivers/gpu/drm/imx/dcss/dcss-crtc.c
+++ b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
@@ -272,8 +272,6 @@ static int dcss_crtc_init(struct dcss_crtc *crtc,
 		return ret;
 	}
 
-	drm->mode_config.allow_fb_modifiers = true;
-
 	disable_irq(crtc->irq);
 
 	return 0;
-- 
1.7.9.5

